src/domains/ecommerce-app/api/ecommerce-checkout/v2/_openapi.json.tpl

```diff
@@ -351,6 +422,124 @@
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/PaymentInfo"
+            },
+            "minItems": 1,
+            "maxItems": 1,
+            "example": [
+              {
+                "rptId": "77777777777302012387654312384",
+                "paymentToken": "paymentToken1",
+                "reason": "reason1",
+                "amount": 600,
+                "transferList": [
+                  {
+                    "paFiscalCode": "77777777777",
+                    "digitalStamp": false,
+                    "transferCategory": "transferCategory1",
+                    "transferAmount": 500
+                  },
+                  {
+                    "paFiscalCode": "11111111111",
+                    "digitalStamp": true,
+                    "transferCategory": "transferCategory2",
+                    "transferAmount": 100
+                  }
+                ]
+              }
+            ]
+          },
+          "status": {
+            "$ref": "#/components/schemas/TransactionStatus"
+          },
+          "feeTotal": {
+            "$ref": "#/components/schemas/AmountEuroCents"
+          },
+          "authToken": {
+            "type": "string"
+          },
+          "idCart": {
+            "description": "Cart identifier provided by creditor institution",
+            "type": "string",
+            "example": "idCartFromCreditorInstitution"
+          },
+          "clientId": {
+            "description": "transaction client id",
+            "type": "string",
+            "enum": [
+              "IO",
+              "CHECKOUT",
+              "CHECKOUT_CART",
+              "UNKNOWN"
+            ]
+          },
+          "nodeInfo":{
+            "type": "object",
+            "description": "Gateway infos",
+            "properties": {
+              "closePaymentResultError": {
+                "type": "object",
+                "description": "Error details for close payment result",
+                "properties": {
+                  "statusCode": {
+                    "type": "number"
+                  },
+                  "description": {
+                    "type": "string"
+                  }
+                }
+              },
+              "sendPaymentResultOutcome": {
+                "description": "The outcome of sendPaymentResult api (OK, KO, NOT_RECEIVED)",
+                "type": "string",
+                "enum": [
+                  "OK",
+                  "KO",
+                  "NOT_RECEIVED"
+                ]
+              },
+            }
+          },
+          "gatewayInfo":{
+            "type": "object",
+            "description": "Gateway infos",
+            "properties": {
+              "gateway": {
+                "type": "string",
+                "pattern": "XPAY|VPOS|NPG|REDIRECT",
+                "description": "Pgs identifier"
+              },
+              "gatewayAuthorizationStatus": {
```

Comment:
```suggestion
              "authorizationStatus": {
```

since this field is not subfield of gatewayInfo we can get rid of the gateway name prefix here

Reply:
done here 56610dd06b7f9d946909bb116733314b58ea209e